src/debugpy/_vendored/pydevd/pydev_ipython/matplotlibtools.py

```diff
@@ -140,11 +140,11 @@ def activate_matplotlib(enable_gui_function):
     if is_interactive:
         enable_gui_function(gui)
         if not matplotlib.is_interactive():
-            sys.stdout.write("Backend %s is interactive backend. Turning interactive mode on.\n" % backend)
+            sys.stderr.write("Backend %s is interactive backend. Turning interactive mode on.\n" % backend)
```

Comment:
I'm thinking this shouldn't write to either. It should probably write to the log instead. 

That would be something like this instead:

```suggestion
            pydev_log.debug("Backend %s is interactive backend. Turning interactive mode on.\n" % backend)
```

Reply:
Of course after importing pdev_log. Maybe that causes some import problems though and it's why this code was writing to stdout.

Reply:
I changed it to `stderr` because I saw it writing to `stderr` in other places in the same file, but logging is fine to me, as long as it does not end in `stdout`.

Reply:
Yeah not sure why it's writing to stderr instead of log elsewhere. I believe this code was originally a port from somewhere else and maybe whomever ported it didn't bother to make it write to the log instead.